R/AllGenerics.R

```diff
@@ -493,3 +493,7 @@ fortify.CrunchDataFrame <- function(model, data, ...) model
 #' @rdname fortify
 #' @export fortify.CrunchDataset
 fortify.CrunchDataset <- function(model, data, ...) model
+
+setGeneric("sendCrunchAutomationRequest", function(x, ...){
```

Comment:
I worry `Request` is an overloaded with meaning when it comes to web traffic. Maybe `sendCrunchAutomationScript()`?